stories/list/object/object-list.stories.js

```diff
@@ -0,0 +1,900 @@
+    </div>
+    </div>
+    <div class="fd-list__object fd-list__object--bottom-row">
+    <div class="fd-list__object fd-list__object--bottom-left">
+      <span class="fd-list-attribute">Second Attribute</span>
+    </div>
+    <div class="fd-list__object fd-list__object--bottom-right">
+    <span class="fd-object-status fd-object-status--inverted fd-object-status--critical">Inverted Warning</span>
+    </div>
+    </div>
+    <div class="fd-list__object fd-list__object--bottom-row">
+    <div class="fd-list__object fd-list__object--bottom-left">
+      <span class="fd-list-attribute">Third Attribute</span>
+    </div>
+    </div>
+    </div>
+    </div>
+  </li>
+  <li role="option" tabindex="0" class="fd-list__item">
+  <div class="fd-list__content">
+     <div class="fd-list-optional">
+     <span class="fd-list-optional__text">Optional inline text</span>
+     </div>
+     <div class="fd-list__object fd-list__object--top-row">
+     <span class="fd-avatar fd-avatar--s" style="background-image: url('http://lorempixel.com/400/400/nature/4/')"></span>
+     <div class="fd-list__object fd-list__object--top-left">
+     <div class="fd-object-identifier">
+       <p class="fd-object-identifier__title">
+     Fitbit Versa Smart Watch, Black Fitbit Versa Smart Watch
+      </p>
+      </div></div>
+     <div class="fd-list__object fd-list__object--top-right fd-object-number fd-object-number--positive">
+     <span class="fd-object-number__text">956.00</span>
+     <span class="fd-object-number__unit">Euro</span>
+     </div>
+    </div>
+    <div class="fd-list__object fd-list__object--bottom">
+    <div class="fd-list__object fd-list__object--bottom-row">
+    <div class="fd-list__object fd-list__object--bottom-right">
+    <span class="fd-object-status fd-object-status--positive">
+    <span class="fd-object-status__text">First status</span></span>
+    </div>
+    </div>
+    <div class="fd-list__object fd-list__object--bottom-row">
+    <div class="fd-list__object fd-list__object--bottom-left">
+      <span class="fd-list-attribute">Second Attribute</span>
+    </div>
+    <div class="fd-list__object fd-list__object--bottom-right">
+    <span class="fd-object-status fd-object-status--positive">
+   <span class="fd-object-status__text">Second status</span></span>
+    </div>
+    </div>
+    <div class="fd-list__object fd-list__object--bottom-row">
+    <div class="fd-list__object fd-list__object--bottom-left">
+      <span class="fd-list-attribute">Third Attribute</span>
+    </div>
+    </div>
+    </div>
+    </div>
+  </li>
+</ul>
+`;
+
+object.storyName = 'Object';
+
+object.parameters = {
+    docs: {
+        iframeHeight: 785
+    }
+};
+
+export const navigation = () => `
+
+<h4 id="objectListItemNavigation">Object List Item With Navigation</h4>
+
+<div style="max-width: 450px">
+<ul class="fd-list fd-list--object fd-list--navigation" role="listbox" aria-labelledby="object List Item With Navigation">
```

Comment:
`aria-labelledby` value should be the `id` of the header above for all `<ul>`s
```suggestion
<ul class="fd-list fd-list--object fd-list--navigation" role="listbox" aria-labelledby="objectListItemNavigation">
```

Reply:
updated